### ISSUE TITLE
Support TextMeshProUGUI instead of Unity Text

### ISSUE THREAD

**Issue by Haapavuo** [roguecode/Unity-Simple-SRT#7]
Should replace Unity Text with TextMeshProUGUI.

**Comment by alexandergovoni** [comment]
I’d also like to have it to work with TMP. I’ve been trying to change the SubtitleDisplayer script but without success. Any tricks you could maybe share?

**Comment by yoiang** [comment]
@akshaybharwani created a pull request #9 that adds Text Mesh Pro support!

**Comment by akshaybharwani** [comment]
Yeah I hope @roguecode would merge with the main. It works perfectly.
